src/services/LoggerService.ts

```diff
@@ -170,14 +170,8 @@ export default class LoggerService extends DependencyAwareClass {
       Sentry.captureException(error);
     }
 
-    if (
-      typeof process.env.EPSAGON_TOKEN === 'string'
-      && process.env.EPSAGON_TOKEN !== 'undefined'
-      && typeof process.env.EPSAGON_SERVICE_NAME === 'string'
-      && process.env.EPSAGON_SERVICE_NAME !== 'undefined'
-      && error instanceof Error
-    ) {
-      Epsagon.setError(error);
+    if (process.env.LUMIGO_TRACER_TOKEN && error instanceof Error) {
```

Comment:
If Lumigo auto-tracing is enabled, am I right in thinking `LUMIGO_TRACER_TOKEN` will *not* be set? Therefore `lumigo.error()` will not be called while auto-tracing?

Reply:
Thanks for taking a look at this @zhibek. Auto-tracing _does_ add the `LUMIGO_TRACER_TOKEN` env var to the Lambda function, so using this env var name means Lambda Wrapper's Lumigo integration should work automatically with auto-traced functions. (Not realising this, I had originally chosen `LUMIGO_TOKEN` as the name; see 4ad8ad0)

I was trying to find a clear reference to this in their documentation. There's nothing that explicitly states it, however on their [Lambda Layers](https://docs.lumigo.io/docs/lambda-layers) page it shows the steps to manually configure the layer, and says something to the effect that auto-trace does this for you. Step 5 there is setting `LUMIGO_TRACER_TOKEN`.

In the readme I said "_Lumigo integration works out-of-the-box with Lumigo's auto-trace feature_". I think this is all that's needed here – if you're using auto-tracing, you don't need to do anything, and you can stop reading. I will target the in-code comment though to explain why the env var check works in both manual and auto mode.